intellij-plugin-verifier/verifier-cli/src/main/java/com/jetbrains/pluginverifier/options/OptionsParser.kt

```diff
@@ -54,14 +56,44 @@ object OptionsParser {
     println("Verification reports directory: $verificationReportsDirectory")
     val teamCityLog = if (opts.needTeamCityLog) TeamCityLog(System.out) else null
     val previousTcHistory = opts.previousTcTestsFile?.let { Paths.get(it) }?.let { TeamCityHistory.readFromFile(it) }
+    val outputFormats = parseOutputFormats(opts)
     return OutputOptions(
       verificationReportsDirectory,
       teamCityLog,
       TeamCityResultPrinter.GroupBy.parse(opts.teamCityGroupType),
-      previousTcHistory
+      previousTcHistory,
+      outputFormats
     )
   }
 
+  private fun parseOutputFormats(opts: CmdOpts): List<OutputFormat> {
+    val (exclusions, inclusions) = opts.outputFormats.partition { it.trim().startsWith("-") }
+
+
+    val includedFormats = inclusions.mapNotNull {
+      try {
+        OutputFormat.valueOf(it.uppercase())
+      } catch (e: IllegalArgumentException) {
+        LOG.warn("Unsupported verification report output format '$it'. Skipping")
```

Comment:
IMHO we should fail execution instead if we can't parse args
not sure log is always visible to user

Reply:
There is always a fallback on default format to maintain backwards compatibility, even in case of failures or syntax errors. This message just logs the fallback.

Furthermore, the log is always visible and enabled for `cli`. This is provided by [explicit configuration of Logback](https://github.com/JetBrains/intellij-plugin-verifier/blob/master/intellij-plugin-verifier/verifier-cli/src/main/resources/logback.xml).